### DISCUSSION TITLE
forgot root password for dja0231 - Telstra Australia

### DISCUSSION THREAD

**Discussion by demiGod095** [hack-technicolor/hack-technicolor#234] [Q&A]
Title says it all.

I still have access to the web ui.
What would be the easiest way to start fresh ?

Edit: I am on the latest rootable firmware listed so, I tried root strategy `#3` but the DHCP request never arrived, I waited for over an hour..
here's the logs as seen on the web interface:
<html><body>

Date | Facility | Process | Message
-- | -- | -- | --
Apr 11 22:37:06 | daemon.notice | wansensing | The L3 DHCP Sense script is sensing DHCP on l2type interface ETH
Apr 11 22:37:06 | daemon.notice | wansensing | (L3DHCPSense) runs L3DHCPSenseMain.check(ETH,timeout)
Apr 11 22:37:06 | daemon.warn | [10273] | Use tch.logger in /usr/bin/atmping:18
Apr 11 22:37:06 | daemon.info | transformer[4810] | async  run: start-stop-daemon -K -p /var/run/atmpinginvoker.pid;  start-stop-daemon -S -b -p /var/run/atmpinginvoker.pid -m -x  /usr/bin/atmpinginvoker
Apr 11 22:37:05 | daemon.warn | mobiled | Failed to retrieve ICCID
Apr 11 22:37:05 | user.notice | cwmpd | waiting for wan6 up ...
Apr 11 22:37:02 | daemon.warn | [10251] | Use tch.logger in /usr/bin/atmping:18
Apr 11 22:37:02 | daemon.info | transformer[4810] | async  run: start-stop-daemon -K -p /var/run/atmpinginvoker.pid;  start-stop-daemon -S -b -p /var/run/atmpinginvoker.pid -m -x  /usr/bin/atmpinginvoker
Apr 11 22:37:01 | daemon.notice | wansensing | (throughput) runs worker_throughput.check(throughput_calculate)
Apr 11 22:37:01 | daemon.notice | wansensing | DHCP check No 39
Apr 11 22:37:01 | daemon.notice | wansensing | The L3 DHCP Sense script is sensing DHCP on l2type interface ETH
Apr 11 22:37:01 | daemon.notice | wansensing | (L3DHCPSense) runs L3DHCPSenseMain.check(ETH,timeout)
Apr 11 22:37:00 | daemon.warn | mobiled | Failed to retrieve ICCID
Apr 11 22:37:00 | user.notice | cwmpd | waiting for wan6 up ...
Apr 11 22:36:56 | daemon.notice | wansensing | (throughput) runs worker_throughput.check(throughput_calculate)
Apr 11 22:36:56 | daemon.notice | wansensing | DHCP check No 38
Apr 11 22:36:56 | daemon.notice | wansensing | The L3 DHCP Sense script is sensing DHCP on l2type interface ETH
Apr 11 22:36:56 | daemon.notice | wansensing | (L3DHCPSense) runs L3DHCPSenseMain.check(ETH,timeout)
Apr 11 22:36:55 | daemon.warn | mobiled | Failed to retrieve ICCID
Apr 11 22:36:55 | user.notice | cwmpd | waiting for wan6 up ...
Apr 11 22:36:51 | daemon.notice | wansensing | (throughput) runs worker_throughput.check(throughput_calculate)
Apr 11 22:36:51 | daemon.notice | wansensing | DHCP check No 37
Apr 11 22:36:51 | daemon.notice | wansensing | The L3 DHCP Sense script is sensing DHCP on l2type interface ETH
Apr 11 22:36:51 | daemon.notice | wansensing | (L3DHCPSense) runs L3DHCPSenseMain.check(ETH,timeout)
Apr 11 22:36:50 | daemon.warn | mobiled | Failed to retrieve ICCID
Apr 11 22:36:50 | daemon.warn | [10172] | Use tch.logger in /usr/bin/atmping:18
Apr 11 22:36:50 | daemon.info | transformer[4810] | async  run: start-stop-daemon -K -p /var/run/atmpinginvoker.pid;  start-stop-daemon -S -b -p /var/run/atmpinginvoker.pid -m -x  /usr/bin/atmpinginvoker
Apr 11 22:36:50 | daemon.warn | [10162] | Use tch.logger in /usr/bin/atmping:18
Apr 11 22:36:50 | daemon.info | transformer[4810] | async  run: start-stop-daemon -K -p /var/run/atmpinginvoker.pid;  start-stop-daemon -S -b -p /var/run/atmpinginvoker.pid -m -x  /usr/bin/atmpinginvoker
Apr 11 22:36:49 | user.notice | cwmpd | waiting for wan6 up ...
Apr 11 22:36:46 | daemon.notice | wansensing | (throughput) runs worker_throughput.check(throughput_calculate)
Apr 11 22:36:46 | daemon.notice | wansensing | DHCP check No 36
Apr 11 22:36:46 | daemon.notice | wansensing | The L3 DHCP Sense script is sensing DHCP on l2type interface ETH
Apr 11 22:36:46 | daemon.notice | wansensing | (L3DHCPSense) runs L3DHCPSenseMain.check(ETH,timeout)
Apr 11 22:36:45 | daemon.warn | mobiled | Failed to retrieve ICCID
Apr 11 22:36:44 | user.notice | cwmpd | waiting for wan6 up ...
Apr 11 22:36:41 | daemon.notice | wansensing | (throughput) runs worker_throughput.check(throughput_calculate)
Apr 11 22:36:41 | daemon.notice | wansensing | DHCP check No 35

</body>
</html>

**Comment by FrancYescO** [comment]
redo rooting procedure.

**Reply by demiGod095** [reply]
Ok, I am on the latest firmware, I tried the tch-exploit and the DHCP request never comes in.. any ideas what can be the issue?

```
➜  release sudo ./tch-exploit-linux
Technicolor OpenWRT Shell Unlocker v2.0.2 By BoLaMN

* Connect network cable from your computer to the WAN (red) port of the modem
* Change your computers network card to be a static ip address

  IPv4 Address: 58.162.0.1
  Subnet Mask: 255.255.255.0
  Default Gateway\Router: 58.162.0.1

using interface { name: 'enx9405bb14a023', address: '58.162.0.1', family: 'IPv4' }
listening for cwmp requests at http://58.162.0.1:46435/
Waiting for DHCP request... 0.0.0.0:67
Waiting for DHCPv6 request... :::547
```

**Reply by nextedoff** [reply]
By the way, I've got the same issue right now but on a different modem and absolutely different software

**Reply by FrancYescO** [reply]
Reconfigure your internet on modem to use DHCP, or you should change VLAN ID on the PC if is actually configure to use it on a VLAN ID
